### PR TITLE
Issue#817

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added some differences.
I've added the ATP plan 2 but it has been 2 hours and I'm still getting 404 errors.
So I'll report those 3 differences that I could "see" from images on the text.

Solving issue #817

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/threat-explorer.md [hunk 1]
```diff
@@ -165,3 +165,8 @@ To learn more about roles and permissions, see the following resources:
 - [Permissions in the Office 365 Security &amp; Compliance Center](permissions-in-the-security-and-compliance-center.md)
 - [Feature permissions in Exchange Online](https://docs.microsoft.com/exchange/permissions-exo/feature-permissions)
   
+## Some differences between Real time detection and Threat Exporter.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/threat-explorer.md]:
```suggestion
## Some differences between real-time detections and Threat Explorer
```

File: SecurityCompliance/threat-explorer.md [hunk 2]
```diff
@@ -165,3 +165,8 @@ To learn more about roles and permissions, see the following resources:
...
+
+ - **Real-Time detection report** is shipped with ATP P1 subscription while **Threat Explorer** comes with ATP P2.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/threat-explorer.md]:
```suggestion
 - **Real-time detection** is shipped with ATP Plan 1 subscription while **Threat Explorer** comes with ATP Plan 2.
```

File: SecurityCompliance/threat-explorer.md [hunk 3]
```diff
@@ -165,3 +165,8 @@ To learn more about roles and permissions, see the following resources:
...
+ - **Real-Time detections** report allows you to show up the detections in real-time while threat explorer reports allow you to see more details in a given attack.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/threat-explorer.md]:
```suggestion
 - **Real-time detection** allows you to show up the detections in real-time while threat explorer reports allow you to see more details in a given attack.
```

File: SecurityCompliance/threat-explorer.md [hunk 4]
```diff
@@ -165,3 +165,8 @@ To learn more about roles and permissions, see the following resources:
...
+ - **Threat Explorer** reports offer different sets of reports that enhance your experience respect **Real-Time Detector** reports.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/threat-explorer.md]:
```suggestion
 - **Threat Explorer** reports offer different sets of reports that enhance your experience in relation to **Real-time detection** .
```

**Pull request merged**